examples/getting_started/2_Pipeline.ipynb

```diff
@@ -388,11 +388,13 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "sel1 = agg_d3_d5.where(aggc.sel(cat='d3') == aggc.sel(cat='d5')).astype('uint32')\n",
-    "sel2 =       agg.where(aggc.sel(cat='d3') == aggc.sel(cat='d5')).astype('uint32')\n",
+    "sel1 = agg_d3_d5.where(aggc.sel(cat=\"d3\") == aggc.sel(cat=\"d5\")).fillna(0).astype(\"uint32\")\n",
+    "sel2 = agg.where(aggc.sel(cat=\"d3\") == aggc.sel(cat=\"d5\")).fillna(0).astype(\"uint32\")\n",
```

Comment:
This does not produce the same output:

![image](https://github.com/user-attachments/assets/c5cdd49a-9d8c-43d3-ba08-8ee7a3f0d2ca)

Reply:
This seems to create the same output with no warning:

![image](https://github.com/user-attachments/assets/4f5dd9f6-841e-45ab-89cf-d528ed87051b)